cloudmarker/alerts/emailalert.py

```diff
@@ -68,10 +79,53 @@ def done(self):
             self.body = ''.join(self.stringbuffer)
         message.attach(MIMEText(self.body))
 
-        smtp_connection = smtplib.SMTP(host=self.host, port=self.port)
+        smtp_connection = self.get_connection()
```

Comment:
The variable name `smtp_connection` is slightly non-idiomatic while working with `smtplib` or sockets/connections in general. A more idiomatic name would be just `smtp` or `server`. I suggest renaming this to `smtp`.

Since this variable lives in a very small scope, I think we can make this variable name short and crisp.

Reply:
https://github.com/cloudmarker/cloudmarker/commit/11ff34083ba27900832dc7daf05a72cd2a99caca#diff-28d4faa43f2d620f5a2c54123370ee47R112